Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:3.0.2-alpine
+
+ENV CF_CLI_VERSION v7
+
+RUN wget -O - "https://packages.cloudfoundry.org/stable?release=linux64-binary&source=github&version=${CF_CLI_VERSION}" \
+    | tar -zx -C /usr/local/bin
+
+COPY Gemfile* .
+
+RUN apk update
+RUN apk add libc-dev make gcc
+
+RUN bundle config set --local without 'development'
```

Comment:
is --local required?

Reply:
--local will set the config in the current project root instead of globally on the system. I guess it doesn't matter in the pipeline but it does when running locally.